packages/sdk/deployments/bsc/DotDotLpERC4626_0xf0a2852958aD041a9Fb35c312605482Ca3Ec17ba.json

```diff
@@ -0,0 +1,1234 @@
+{
+  "address": "0x3c29e9b0CfE6FfF97f373eAbEADE9475FaC3bd4e",
```

Comment:
why does the name of this file contain an address suffix?

Reply:
It's the address of the market it was deployed for.